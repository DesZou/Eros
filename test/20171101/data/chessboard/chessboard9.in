8
*oooooo*
*oooooo*
*oooooo*
*oooooo*
*oooooo*
*oooooo*
*oooooo*
*oooooo*
