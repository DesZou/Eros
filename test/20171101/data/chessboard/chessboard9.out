1
92147691
958321673
497436442
362360458
64461920
12160
0
0
